application/app/src/test/java/uk/nhs/digital/uec/api/integration/fuzzysearch/FuzzySearchGeneralErrorsTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.nhs.digital.uec.api.integration.fuzzysearch;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ActiveProfiles({"prod"})
+/**
+ * Test class which passes requests through the Fuzzy Search endpoint and asserts desired API
+ * behavior. Only the model layer will be mocked here.
+ */
+public class FuzzySearchGeneralErrorsTest {
+
+  private static String endpointUrl =
+      "http://localhost:9095/dosapi/dosservices/v0.0.1/services/byfuzzysearch";
```

Comment:
Can we move away from versioning of the API in the code i.e. hardcoded URL to `v0.0.1` in favour of versioning supported by pointing to the right application by means of Route53, API Gateway or Apigee.

Reply:
Yes, once I know how to do that :)
I'll extract it into an environment var for now....

Reply:
Sounds good